2024-04-20 22:41:40.069848: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-04-20 22:41:40.071326: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-04-20 22:41:40.072310: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-04-20 22:41:40.078397: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-20 22:41:41.658955: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
[nltk_data] Downloading package punkt to
[nltk_data]     /home/abdelrahman.sadallah/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
`low_cpu_mem_usage` was None, now set to True since model is quantized.
Loading checkpoint shards:   0%|          | 0/2 [00:00<?, ?it/s]Loading checkpoint shards:  50%|█████     | 1/2 [05:00<05:00, 300.44s/it]Loading checkpoint shards: 100%|██████████| 2/2 [06:14<00:00, 167.12s/it]Loading checkpoint shards: 100%|██████████| 2/2 [06:14<00:00, 187.12s/it]
  0%|          | 0/36 [00:00<?, ?it/s]  3%|▎         | 1/36 [00:43<25:08, 43.09s/it]  6%|▌         | 2/36 [01:35<27:25, 48.39s/it]  8%|▊         | 3/36 [02:37<30:07, 54.78s/it] 11%|█         | 4/36 [03:40<30:49, 57.80s/it] 14%|█▍        | 5/36 [04:42<30:42, 59.45s/it] 17%|█▋        | 6/36 [05:15<25:15, 50.51s/it] 19%|█▉        | 7/36 [05:39<20:09, 41.71s/it] 22%|██▏       | 8/36 [06:22<19:42, 42.24s/it] 25%|██▌       | 9/36 [07:25<21:53, 48.64s/it] 28%|██▊       | 10/36 [08:28<23:00, 53.10s/it] 31%|███       | 11/36 [09:30<23:16, 55.86s/it] 33%|███▎      | 12/36 [10:33<23:10, 57.95s/it] 36%|███▌      | 13/36 [11:21<21:04, 54.99s/it] 39%|███▉      | 14/36 [12:24<21:05, 57.54s/it] 42%|████▏     | 15/36 [12:57<17:29, 49.96s/it] 44%|████▍     | 16/36 [14:01<18:04, 54.23s/it] 47%|████▋     | 17/36 [14:46<16:17, 51.45s/it] 50%|█████     | 18/36 [15:09<12:55, 43.07s/it] 53%|█████▎    | 19/36 [16:13<13:57, 49.28s/it] 56%|█████▌    | 20/36 [16:43<11:36, 43.56s/it] 58%|█████▊    | 21/36 [17:24<10:40, 42.67s/it] 61%|██████    | 22/36 [18:26<11:19, 48.57s/it] 64%|██████▍   | 23/36 [19:28<11:24, 52.65s/it] 67%|██████▋   | 24/36 [20:32<11:10, 55.87s/it] 69%|██████▉   | 25/36 [21:16<09:35, 52.30s/it] 72%|███████▏  | 26/36 [22:20<09:19, 55.97s/it] 75%|███████▌  | 27/36 [23:18<08:27, 56.34s/it] 78%|███████▊  | 28/36 [24:21<07:48, 58.57s/it] 81%|████████  | 29/36 [25:24<06:59, 59.95s/it] 83%|████████▎ | 30/36 [26:27<06:03, 60.65s/it] 86%|████████▌ | 31/36 [26:57<04:16, 51.39s/it] 89%|████████▉ | 32/36 [28:00<03:40, 55.11s/it] 92%|█████████▏| 33/36 [29:03<02:52, 57.47s/it] 94%|█████████▍| 34/36 [30:09<02:00, 60.01s/it] 97%|█████████▋| 35/36 [31:12<01:00, 60.70s/it]100%|██████████| 36/36 [32:21<00:00, 63.21s/it]100%|██████████| 36/36 [32:21<00:00, 53.92s/it]
Some weights of RobertaModel were not initialized from the model checkpoint at roberta-large and are newly initialized: ['roberta.pooler.dense.bias', 'roberta.pooler.dense.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Traceback (most recent call last):
  File "/home/abdelrahman.sadallah/mbzuai/UWFe/evaluate_llm.py", line 293, in <module>
    with open(save_file, 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'outputs/outputs/Mistral-7B-v0.1_metrics_0-shots_True-explicit.txt'
[2024-04-20 23:21:46,462] torch.distributed.elastic.multiprocessing.api: [ERROR] failed (exitcode: 1) local_rank: 0 (pid: 642595) of binary: /home/abdelrahman.sadallah/mambaforge/envs/nlp/bin/python
Traceback (most recent call last):
  File "/home/abdelrahman.sadallah/mambaforge/envs/nlp/bin/torchrun", line 8, in <module>
    sys.exit(main())
  File "/home/abdelrahman.sadallah/mambaforge/envs/nlp/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 347, in wrapper
    return f(*args, **kwargs)
  File "/home/abdelrahman.sadallah/mambaforge/envs/nlp/lib/python3.10/site-packages/torch/distributed/run.py", line 812, in main
    run(args)
  File "/home/abdelrahman.sadallah/mambaforge/envs/nlp/lib/python3.10/site-packages/torch/distributed/run.py", line 803, in run
    elastic_launch(
  File "/home/abdelrahman.sadallah/mambaforge/envs/nlp/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 135, in __call__
    return launch_agent(self._config, self._entrypoint, list(args))
  File "/home/abdelrahman.sadallah/mambaforge/envs/nlp/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 268, in launch_agent
    raise ChildFailedError(
torch.distributed.elastic.multiprocessing.errors.ChildFailedError: 
============================================================
evaluate_llm.py FAILED
------------------------------------------------------------
Failures:
  <NO_OTHER_FAILURES>
------------------------------------------------------------
Root Cause (first observed failure):
[0]:
  time      : 2024-04-20_23:21:46
  host      : gpu-10
  rank      : 0 (local_rank: 0)
  exitcode  : 1 (pid: 642595)
  error_file: <N/A>
  traceback : To enable traceback see: https://pytorch.org/docs/stable/elastic/errors.html
============================================================
